src/ring_theory/little_wedderburn.lean

```diff
@@ -0,0 +1,217 @@
+/-
+Copyright (c) 2021 Johan Commelin. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Johan Commelin
+-/
+import group_theory.group_action.class_formula
+import group_theory.group_action.conj_act
+import ring_theory.polynomial.cyclotomic.eval
+import tactic.by_contra
+
+/-!
+# Little Wedderburn TODO
```

Comment:
This is done, isn't it?

Reply:
I think the TODO is writing the documentation - I'll get on that soon!